docs/UserGuide.md

```diff
@@ -132,6 +136,130 @@ Examples:
 * `addLocation idl/L123 n/Vivocity a/John street, block 123, #01-01`
 * `addLocation idl/L234 n/Betsy Crowe's House a/Newgate Prison`
 
+#### Adding data from CSV files
+
+As you may have pre-existing data stored in the Excel file format, VirusTracker provides a way to import data directly from
+files in the CSV format. Excel provides an option to save existing _.xlsx_ extension files as _.csv_ files.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Note:**
+If you are importing data from a pre-existing Excel file, you may have to first format it to a format that is readable by VirusTracker.
+</div>
+
+You may read more about it [here](#format-for-csv-files).
+
+Format: `addFromCsv FILE_PATH l/LIST_TYPE`
+
+* `FILE_PATH` refers to the absolute file path where the CSV file resides.
+  * For example, if you wish to import data from `personList.csv` located in your desktop, the absolute file path could look
+  something like this: `C:/Users/user/Desktop/personList.csv` _(for Windows)_
```

Comment:
Consider providing the examples for Linux as well? Since target users may be using linux too.

Reply:
Done